active_learning/tool_tracking_al/results_uncertainty_analysis.ipynb

```diff
@@ -184,11 +184,7 @@
     "device = torch.device(\"cuda\")\n",
```

Comment:
<p>Should this figure be retained and only remove the print info? Not a block problem for now, feel free to fix it later.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1391/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
If we keep that figure it would be incosistent with the latest result/plot that is shared in the Readme.